admin/class-admin-apple-news.php

```diff
@@ -135,13 +135,21 @@ public function __construct() {
 			add_action(
 				'rest_api_init',
 				function() {
-					register_rest_field(
-						'post',
-						'apple_news_notices',
-						[
-							'get_callback' => [ 'Admin_Apple_Notice', 'get_if_allowed' ],
-						]
-					);
+					$admin_settings = new Admin_Apple_Settings();
+					$post_types     = $admin_settings->fetch_settings()->post_types ?? [];
```

Comment:
This class has a static property called `$settings` that contains the result of `fetch_setings()` on this class, so this block can be simplified. Also, we can't use the null coalescing operator because the plugin supports down to PHP 5.6.
```suggestion
					$post_types = ! empty( self::$settings->post_types ) ? self::$settings->post_types : [];
```